editor/plugins/skeleton_editor_plugin.h

```diff
@@ -35,11 +35,87 @@
 #include "editor/editor_plugin.h"
 #include "scene/3d/skeleton.h"
 
-class PhysicalBone;
+class EditorInspectorPluginSkeleton;
 class Joint;
+class PhysicalBone;
+class SkeletonEditorPlugin;
+class Button;
+class CheckBox;
+
+class BoneTransformEditor : public VBoxContainer {
+	GDCLASS(BoneTransformEditor, VBoxContainer);
+
+	EditorInspectorSection *section;
+
+	GridContainer *translation_grid;
+	GridContainer *rotation_grid;
+	GridContainer *scale_grid;
+	GridContainer *transform_grid;
+
+	EditorSpinSlider *translation[3];
+	EditorSpinSlider *rotation[3];
+	EditorSpinSlider *scale[3];
+	EditorSpinSlider *transform[12];
```

Comment:
The name of these are miss leading; Who checked the code for the first time will find strange that it's possible to set the stepping or the colour to a variable called `translation`.

I propose to be more precise, so something like would be better:
```cpp
EditorSpinSlider *translation_slider[3];
EditorSpinSlider *rotation_slider[3];
EditorSpinSlider *scale_slider[3];
EditorSpinSlider *transform_slider[12];
```
or `spin` or `input`, or anything that tells that it's not just a `Vector3`

Reply:
Renamed.